Formula/triangle.rb

```diff
@@ -0,0 +1,26 @@
+class Triangle < Formula
+  desc "Convert images to computer generated art using delaunay triangulation"
+  homepage "https://github.com/esimov/triangle"
+  url "https://github.com/esimov/triangle/archive/v1.0.3.tar.gz"
+  sha256 "0130bdc9d6c6dd35234134dcb97885d48fa22b5c7532767efbe0394497b96fdd"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = bin
+
+    dir = buildpath/"src/github.com/esimov/triangle"
+
+    dir.install buildpath.children
+
+    cd dir/"cmd/triangle" do
+      system "go", "install"
```

Comment:
Does this include or download any resources?

Reply:
Probably not. The archive contains `vendor` directory for dependencies.
(I can successfully install the formula without an internet connection.)